src/utils.js

```diff
@@ -71,3 +73,5 @@ export const setByPath = (obj, path = [], value) => {
 export const arrayFromAllowNullOrUndefined = arrayLike => (
   isNullOrUndefined(arrayLike) ? [] : [...arrayLike]
 );
+
+export const getRandomId = () => Math.random().toString(16).slice(2);
```

Comment:
can we change it to `getNextId`? 

- If id is incremental, it is easier for us to tell which listener bind first and which comes later? 
- no way to conflict.